lte/gateway/python/integ_tests/federated_tests/README.md

```diff
@@ -6,87 +6,93 @@ hide_title: true
 # Federated Integration Tests (EXPERIMENTAL)
 
 The objective of Federated Integration Tests is to provide a test platform
-where to run Magma with all its components.
+to run Magma with all its components.
 
-As the diagram indicates, this test spins up AGW, FEG and Orc8r
-and uses S1aP and trf server to run tests.
+As the diagram indicates, this test spins up **AGW**, **FeG**, and **Orc8r**
+and uses `S1aP` and `magma_trfserver` as an eNb and SGi emulator to run tests.
 
 ```mermaid
 graph LR;
-    S1AP --> AGW --> orc8r --> FEG --> 3ggp_CORE;
-    3ggp_CORE --> FEG --> orc8r --> AGW --> S1AP;
-    AGW --> trf_server;
-    trf_server --> AGW;
+    A[[S1AP]] --- AGW ---|control| orc8r --- FEG --- HSS[(HSS)];
+    AGW ---|data| Z[[magma_trfserver]];
```

Comment:
Wow that's so pretty / looks so easy to write. Do you have a pointer to the doc/tutorial you used to understand mermaid?

Reply:
https://mermaid-js.github.io/mermaid/#/./flowchart?id=flowcharts-basic-syntax